app/controllers/items_controller.rb

```diff
@@ -0,0 +1,51 @@
+class ItemsController < ApplicationController
+  def index
+    if params[:merchant_id]
+      @merchant = Merchant.find(params[:merchant_id])
+      @items = @merchant.items
+    else
+      @items = Item.all
+    end
+  end
+
+  def show
+    @item = Item.find(params[:id])
+  end
+
+  def new
+    @merchant = Merchant.find(params[:merchant_id])
+  end
+
+  def create
+    merchant = Merchant.find(params[:merchant_id])
+    item = merchant.items.create!(item_params)
+    item.active = true
+    redirect_to "/merchants/#{merchant.id}/items"
+  end
+
+  def edit
+    @item = Item.find(params[:id])
+  end
+
+  def update
+    @item = Item.find(params[:id])
+
+    @item.update(item_params)
+    redirect_to "/items/#{@item.id}"
+  end
+
+  def destroy
+    @item = Item.find(params[:id])
+    # binding.pry
+    @item.destroy
+    redirect_to "/items"
+  end
+
+  private
+
+def item_params
```

Comment:
If you were to pry into this method, you can see that the params that you have in your response does not have :active. For this reason, you don't need to permit :active when creating an item. You can still keep what you have in the create method with `item.active = true` and that will define that attribute for you since your items table already knows that you have an attribute called active. The other option you could have looked into is having a default value for active once an item is created. That way, you wouldn't need to define it in the create method.